spec/models/category_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :model do
+  subject do
+    @user = User.new(name: 'Stephen', email: '123@mail.com', password: 'password')
+    @group = @user.categories.new(name: 'Travel', icon: 'https://pxhere.com/en/photo/1569711', user_id: @user.id)
+  end
+  before { subject.save }
+
```

Comment:
- [ ] Good job adding unit tests for the models. Please also add integration tests for the app components.

Reply:
Done
Integration test added
Thanks!